src/main.cpp

```diff
@@ -6289,10 +6289,11 @@ bool ProcessMessages(CNode* pfrom, CConnman& connman)
         uint256 hash = Hash(vRecv.begin(), vRecv.begin() + nMessageSize);
         if (memcmp(hash.begin(), hdr.pchChecksum, CMessageHeader::CHECKSUM_SIZE) != 0)
         {
-            LogPrintf("%s(%s, %u bytes): CHECKSUM ERROR expected %s was %s\n", __func__,
+            LogPrintf("%s(%s, %u bytes): CHECKSUM ERROR expected=%s was=%s peer=%d\n", __func__,
                SanitizeString(strCommand), nMessageSize,
+               HexStr(hdr.pchChecksum, hdr.pchChecksum+CMessageHeader::CHECKSUM_SIZE),
```

Comment:
@paveljanik I see what you are saying now. Ok, so "was" and "expected" are both ambiguous it seems. Perhaps "declared" and "calculated"?